GooglePayment/src/main/java/com/braintreepayments/api/GooglePayment.java

```diff
@@ -151,7 +150,7 @@ public void onComplete(@NonNull Task<Boolean> task) {
      * {@link CardRequirements} via
      * {@link CardRequirements.Builder#addAllowedCardNetworks(Collection)}}.
      *
-     * @param fragment {@link BraintreeFragment}
+     * @param fragment {@link com.braintreepayments.api.BraintreeFragment}
```

Comment:
```suggestion
     * @param fragment {@link BraintreeFragment}
```
I think we decided that adding the full package doesn't do anything.

Reply:
Yeah good catch! I left this line by mistake.